src/core/installAddon.js

```diff
@@ -333,6 +334,23 @@ export class WithInstallHelpers extends React.Component {
     }
   }
 
+  isAddonEnabled() {
+    const {
+      _addonManager,
+      addon: { guid },
+    } = this.props;
+
+    log.info('Getting add-on enabled status');
+    return _addonManager
+      .getAddon(guid)
+      .then((addon) => {
+        return addon.isEnabled;
+      })
+      .catch((error) => {
+        log.error(`Caught error from addonManager: ${error}`);
```

Comment:
Should we return `false`?

Reply:
Can we move to async/await syntax?

Reply:
> Can we move to async/await syntax?

that would be cool : ) i actually thought about that but was trying to keep consistent with other funcs..